Makefile

```diff
@@ -48,7 +48,8 @@ check-manifest:
 	check-manifest
 
 check-distutils:
-	python setup.py check --restructuredtext --strict
+	echo "WARNING: 'make check-distutils' is disabled!!"
+	# python setup.py check --restructuredtext --strict
```

Comment:
Just adding a comment here to make sure we don't miss this, it will obviously be outdated when someone fixes it.

So in the same way as you changed the other commands, how do we run this command in the new format? If you can tell me on the command line, I should be able to make it work in the Makefile, i.e. I think it's a Python not a Make issue.

This page explains what it did, the correct answer might be that these particular commands are no longer required, but hopefully there's a similar set of tools to validate pyproject.toml we should switch to instead? It's interesting that we weren't validating the metadata itself before!